src/test_l3_dsVideoDevice.c

```diff
@@ -368,13 +356,12 @@ void test_l3_dsVideoDevice_SetDisplayFramerate()
     gTestID = 3;
     UT_LOG_INFO("In %s [%02d%03d]", __FUNCTION__, gTestGroup, gTestID);
     dsError_t status   = dsERR_NONE;
-    char frameRate[100] = {0};
-    int32_t j = 0;
+    char frameRate[DS_FRAMERATE_KEY_SIZE] = {0};
 
     dsVideoDevice_getHandle();
     
-    UT_LOG_MENU_INFO(" Provided Display Framerate :");
-    scanf("%s", &frameRate);
+    UT_LOG_MENU_INFO(" Provide Display Framerate :");
```

Comment:
check is same used in the class `Provide Display Framerate `

Reply:
Updated the same in class file as well.